module/armor/basic_armor.cpp

```diff
@@ -456,13 +456,12 @@ cv::Mat Detector::bgrPretreat(cv::Mat&  _src_img,
   cv::split(_src_img, _split);
 
   std::string window_name = {"[basic_armor] brgPretreat() -> color_trackbar"};
-  cv::namedWindow(window_name);
-
   switch (_my_color) {
-    case 0:
+    case uart::BLUE:
       cv::subtract(_split[0], _split[2], bin_color_img);
 
-      if (image_config_.color_edit) {
+      if (image_config_.color_edit == 1) {
+        cv::namedWindow(window_name);
         cv::createTrackbar("blue_color_th", window_name,
                            &image_config_.blue_armor_color_th, 255, NULL);
         cv::imshow(window_name, this->bgr_trackbar_);
```

Comment:
```suggestion
        cv::imshow(window_name, bgr_trackbar_);
```
不需要刻意显式地使用 `this`，留给编译器去做这件事